xbmc/music/ContextMenus.cpp

```diff
@@ -44,24 +44,26 @@ bool CMusicInfo::Execute(const std::shared_ptr<CFileItem>& item) const
 
 bool CMusicBrowse::IsVisible(const CFileItem& item) const
 {
-  if (item.IsFileFolder(EFILEFOLDER_MASK_ONBROWSE))
-    return false; // handled by CMediaWindow
-
-  return item.m_bIsFolder && MUSIC_UTILS::IsItemPlayable(item);
+  return ((item.m_bIsFolder || item.IsFileFolder(EFILEFOLDER_MASK_ONBROWSE)) &&
+          MUSIC_UTILS::IsItemPlayable(item));
 }
 
 bool CMusicBrowse::Execute(const std::shared_ptr<CFileItem>& item) const
 {
+  // For file directory browsing, we need item's dyn path, for everything else the path.
```

Comment:
Care to be explain what is "everything else"?

Reply:
It is just what `CMusicBrowse::IsVisible` returns true for, except browsable file directories.

If you want, I can add this to the comment, but don't think this is needed, because impl of `CMusicBrowse::IsVisible` is just some lines above.

Reply:
I think the comment is wrong and you actually mean the opposite. Mind checking?

Reply:
> I think the comment is wrong and you actually mean the opposite. Mind checking?

Cannot see a problem here...

```c++
  // For file directory browsing, we need item's dyn path, for everything else the path.
  const std::string path{item->IsFileFolder(EFILEFOLDER_MASK_ONBROWSE) ? item->GetDynPath()
                                                                       : item->GetPath()};
```

The comment is correct, I do not mean the opposite.